src/main/java/com/pusher/client/channel/impl/PrivateEncryptedChannelImpl.java

```diff
@@ -68,7 +67,7 @@ public void bind(final String eventName, final SubscriptionEventListener listene
         } catch (final Exception e) {
             // any other errors need to be captured properly and passed upwards
             throw new AuthorizationFailureException("Unable to parse response from Authorizer: "
```

Comment:
```suggestion
            throw new AuthorizationFailureException("Unable to parse response from Authorizer", e);
```
could be one line now (fits 100 char limit), and `: ` could be a bit misleading as we're no longer concatenating it with the actual response (minor, and needs to be applied manually to compile as GH doesn't allow to suggest on removed lines)

Reply:
Hmm. I've updated it.